docs/validators/staking/merge-validator-delegation-sc.md

```diff
@@ -47,3 +47,27 @@ Whitelist Wallet For Merging
 :::warning
 We advise against using this method to buy or sell validator slots - it requires the transfer of private keys (validatorKey.pem) which can't be changed. This puts the buyer at risk of slashing, should the seller deploy a node with the same key, either intentionally or by mistake.
 :::
+
+## **Merging own node(s)**
+
+If the owner address of the node(s) and Delegation SC is the same use, whitelisting is not needed.
+Send the following transaction :
+
+```rust
```

Comment:
lines 56-61 can be removed since the information is duplicated. Please apply the same suggestion to the above section as well.

Reply:
:+1: